tools/opensnoop.bt

```diff
@@ -41,7 +35,12 @@
  * 08-Sep-2018	Brendan Gregg	Created this.
  */
 
-config = { missing_probes=warn }
+#define MAX_PATH_DEPTH	35
```

Comment:
This is only used in one place, why have a define here? Or if you want it configurable how about `getopt` ?